config/test.config

```diff
@@ -11,7 +11,6 @@
         ]
     },
     {grpcbox, [
-%%            {client, #{channels => [{default_channel, [{http, "localhost", 10001, []}], #{}}]}},
             {servers,
```

Comment:
if the whole grpcbox config block is being removed from the main sys.config does the test.config file still need any of that config as well?

Reply:
should be fine to remove, sibyl_ct_utils  sets the config per node

Reply:
as per prev comment still required in test config